src/detext/run_detext.py

```diff
@@ -71,7 +71,7 @@ def add_arguments(parser):
                         help="Whether to use normalized lm. This option only works for unit_type=lstm_lm")
 
     # Optimizer
-    parser.add_argument("--optimizer", type=str, choices=["sgd", "adam", "bert_adam"], default="sgd",
+    parser.add_argument("--optimizer", type=str, choices=["sgd", "adam", "bert_adam", "bert_lamb"], default="sgd",
```

Comment:
As we have more choices for the optimizer, it'll be great if we can have a section on the optimizers in the training instruction

Reply:
I see. Do you mean in the TRAINING.md? I can add some reference for the different optimizers in the parameter table.